adapters/ion-adapter/pom.xml

```diff
@@ -0,0 +1,149 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <parent>
+    <artifactId>adapters</artifactId>
+    <groupId>replication</groupId>
+    <version>0.3.0-SNAPSHOT</version>
+  </parent>
+  <modelVersion>4.0.0</modelVersion>
+
+  <artifactId>ion-adapter</artifactId>
+  <name>Replication :: Adapters :: ION</name>
+  <groupId>replication-adapters</groupId>
+  <packaging>bundle</packaging>
+
+  <dependencies>
+    <dependency>
+      <groupId>replication</groupId>
+      <artifactId>replication-api</artifactId>
+      <version>${project.version}</version>
+    </dependency>
+
+    <dependency>
+      <groupId>org.springframework</groupId>
+      <artifactId>spring-web</artifactId>
```

Comment:
Why did we need to bring in spring web, spring context, and springframework.test?

Reply:
We are using springs RestTemplate which requires spring-web and spring-core. The context one was needed for the service annotations. The test dependency is for some spring test classes I used for testing.